sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -453,6 +453,7 @@ def _receive( # pylint:disable=unused-argument
                 on_error=on_error,
                 on_partition_initialize=on_partition_initialize,
                 on_partition_close=on_partition_close,
+                **kwargs
```

Comment:
What's being passed here to EventProcessor? and is it being used there?

Reply:
oh this I believe is a merge conflict can remove, nothing is passed :)